website/src/content/docs/docs/handbook/breaking-change-policy.mdx

```diff
@@ -0,0 +1,54 @@
+  documented
+
+| Example                          | Level                               | Description                                                                                                                                                                                                                                                                                       |
+| -------------------------------- | ----------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TypeKit                          | <Badge text="0" variant="danger" /> | TypeKit will be moving forward the recommend way to expose apis for a library. Those should be extremely stable.                                                                                                                                                                                  |
+| Additional helpers for a library | <Badge text="0" variant="danger" /> | Depending on the helper this could be considered critical or more advanced usage                                                                                                                                                                                                                  |
+| TypeSpec SyntaxTree              | <Badge text="2" variant="tip" />    | The TypeSpec Ast is not part of the contract a library should depend on.                                                                                                                                                                                                                          |
+| Checker                          | <Badge text="2" variant="tip" />    | Checker is an internal of the typespec program                                                                                                                                                                                                                                                    |
+| Symbols                          | <Badge text="2" variant="tip" />    | Internals                                                                                                                                                                                                                                                                                         |
+| Formatter                        | <Badge text="1" variant="note" />   | Formatting shouldn't' affect the meaning of the language. Changes to the formatting should be taken with consideration as it could create a lot of diff but it shouldn't also be considered a breaking change                                                                                     |
```

Comment:
```suggestion
| Formatter                        | <Badge text="1" variant="note" />   | Formatting shouldn't affect the meaning of the language. Changes to the formatting should be taken with consideration as it could create a lot of diff but it shouldn't also be considered a breaking change                                                                                     |
```